modules/ossm-vs-istio.adoc

```diff
@@ -89,3 +89,8 @@ spec:
 = The Istio Container Network Interface (CNI) plug-in
 
 {ProductName} includes CNI plug-in, which provides you with an alternate way to configure application pod networking. The CNI plug-in replaces the `init-container` network configuration eliminating the need to grant service accounts and projects access to Security Context Constraints (SCCs) with elevated privileges.
+
+= Envoy, Secret Discovery Service, and Certificates
+
+* The QUIC listener for Envoy is not available in {ProductName}. 
+* Deployment of TLS certificates using the Secret Discovery Service (SDS) functionality of Istio is not currently supported in {ProductName}. The Istio implementation depends on a nodeagentcontainer that uses hostPath mounts.
```

Comment:
missing space: `nodeagent container`

Reply:
now that I'm reading it again, maybe add at the end: `that uses hostPath mounts, which are incompatible with the multitenancy feature of {ProductName}.` (that last part sounds a bit dodgy still, not sure how to better formulate it)